modules/node/src/transfer/transfer.service.ts

```diff
@@ -216,10 +216,14 @@ export class TransferService {
           "DEPOSIT_CONFIRMED_EVENT",
           async (msg: DepositConfirmationMessage) => {
             if (msg.from !== this.cfCoreService.cfCore.publicIdentifier) {
-              return;
+              reject(
+                `Deposit event from field: ${msg.from}, did not match public identifier: ${this.cfCoreService.cfCore.publicIdentifier}`,
+              );
```

Comment:
we don't really want to reject here, since you could be getting a `DEPOSIT_CONFIRMED_EVENT` from collateralizing other channels, or in this case from the user of other channels